.pre-commit-config.yaml

```diff
@@ -37,7 +37,7 @@ repos:
         args:
           - --config=.mdl_config.yaml
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.5.1
+    rev: v2.6.1
```

Comment:
We generally let these sorts of improvements flow down from [cisagov/skeleton-packer](https://github.com/cisagov/skeleton-packer).

Reply:
@jsf9k Understood however I was running into pre-commit issues in another repo: https://github.com/cisagov/con-pca-web/pull/327#issuecomment-1084930842

I figured since there was pre-commit issues elswhere i'd run `pre-commit autoupdate` here as well due to the terraform fmt and validate hooks erroring out. It solved the terraform fmt error for me. If you'd like I can reverse versions for the successful hooks.

Reply:
@itsmostafa Yes, please revert that commit. We keep our `pre-commit` configuration in lock-step (barring required per-project modifications) across all of our repositories to ensure that all testing is the same. The updates to the `pre-commit` configuration will get pushed down when https://github.com/cisagov/skeleton-packer/pull/120 is merged.

Reply:
@mcdonnnj i reverted all the pre-commits except for the terraform hook due to this error from the older, existing version:

```
Terraform fmt............................................................Failed
- hook id: terraform_fmt
- exit code: 2

/Users/mostafaabdelbaky/.cache/pre-commit/repoyjmurj0f/hooks/_common.sh: line 30: declare: -g: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
```

Reply:
@mcdonnnj the precommit black hook is also failing:

```
File "/home/runner/.cache/pre-commit/repo5h3gt00v/py_env-python3/bin/black", line 8, in <module>
    sys.exit(patched_main())
  File "/home/runner/.cache/pre-commit/repo5h3gt00v/py_env-python3/lib/python3.9/site-packages/black/__init__.py", line 1[42](https://github.com/cisagov/windows-commando-packer/runs/5824965065?check_suite_focus=true#step:16:42)3, in patched_main
    patch_click()
  File "/home/runner/.cache/pre-commit/repo5h3gt00v/py_env-python3/lib/python3.9/site-packages/black/__init__.py", line 1409, in patch_click
    from click import _unicodefun
ImportError: cannot import name '_unicodefun' from 'click' (/home/runner/.cache/pre-commit/repo5h3gt00v/py_env-python3/lib/python3.9/site-packages/click/__init__.py)
```

Reply:
@itsmostafa Did you rebase on `develop` or merge `develop` into this branch? I'm noticing differences in your `.pre-commit-config.yaml` that should not be there.

Reply:
@mcdonnnj i updated this config file to match what's in develop here: [96d478a](https://github.com/cisagov/windows-commando-packer/pull/10/commits/96d478aa8c4988d7905eb3ea5d1f71a298f3433d)

Reply:
and yes, i merged develop earlier

Reply:
I do not see a recent `develop` merge or force-push to indicate a rebase.

Reply:
@mcdonnnj i tried pushing develop it already is up to date. and I have the latest from develop pulled locally.

```console
➜  windows-commando-packer git:(first-commits) git merge develop
Already up to date.
```

Reply:
@itsmostafa I was able to get things to line up after rebasing on `develop` and dropping a commit. I can force-push my local if you'd like or I can walk you through what I did if you'd prefer to handle it instead.

Reply:
@mcdonnnj If you can walk me through it that would be great. but either works!

Reply:
@mcdonnnj any updates on this? thanks

Reply:
@mcdonnnj this PR is ready for your review. let me know if this can be marked as resolved or if anything additional needs to be done. Thanks!